getting_started.md

```diff
@@ -62,7 +62,7 @@ vcs.download_sample_data_files()
 # The vcs_canvas is the root object of VCS
 vcs_canvas = vcs.init()
 
-cdms_file = cdms2.open(vcs.prefix + "/sample_data/clt.nc")
+cdms_file = cdms2.open(vcs.prefix + "/share/cdat/sample_data/clt.nc")
```

Comment:
really it should
```python
import os
import cdat_info
cdms_file = cdms2.open(os.path.join(cdat_info.get_sampledata_path(), "clt.nc"))
```

Reply:
In trying to test this out, I was not able to import vcs or cdms2 using the cdat8 environment mentioned in the installation instructions. @doutriaux1 if you have time next week, perhaps we can look at this together.

Reply:
Done.